bulk/eng_vocab.csv

```diff
@@ -1,18 +1,18 @@
-DESCRIPTION OF EQUIPMENT,Manufacturer,Model
-Coastal Surface Piercing Profiler Controller,WET Labs,AMP
-Coastal Wire-Following Profiler Controller,McLane,MMP-7
-Communications and Power Manager,WHOI,CPM
-Data Concentrator Logger,WHOI,DCL
-Global Wire-Following Profiler Controller,McLane,MMP 210-3512
-Mobile Asset Controller for Coastal Glider,Teledyne Webb,G2 Slocum Gliders
-Mobile Asset Controller for Profiling Glider,Teledyne Webb,G2 Slocum Gliders
-Mobile Asset Controller for Open Ocean Glider,Teledyne Webb,G2 Slocum Gliders
-Multi-Function Node Communications and Power Manager,WHOI,CPM
-Multi-Function Node Data Concentrator Logger,WHOI,DCL
-Near Surface Instrument Frame Communications and Power Manager,WHOI,CPM
-Near Surface Instrument Frame Data Concentrator Logger,WHOI,DCL
-Sensor and Telemetry Platform Controller,WHOI,STC
-Subsurface Mooring Platform Controller,SIO,Subsurface Mooring Controller
-Surface Buoy Communications and Power Manager,WHOI,CPM
-Surface Buoy Data Concentrator Logger,WHOI,DCL
-Mobile Asset Controller for Coastal AUV,Hydroid,AUV
+DESCRIPTION OF EQUIPMENT,Manufacturer,Model
+Coastal Surface Piercing Profiler Controller,WET Labs,AMP
+Coastal Wire-Following Profiler Controller,McLane,MMP-7
+Communications and Power Manager,WHOI,CPM
+Data Concentrator Logger,WHOI,DCL
+Global Wire-Following Profiler Controller,McLane,MMP 210-3512
+Mobile Asset Controller for Coastal Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Profiling Glider,Teledyne Webb,G2 or G3 Slocum Gliders
+Mobile Asset Controller for Open Ocean Glider,Teledyne Webb,G2 or G3 Slocum Gliders

```

Comment:
addition of 'or G3' looks good

Reply:
Wasn't sure whether to add "or G3" or just remove the model numbers.

Reply:
That seems fine.  We definitely have both, so adding is good.